fog/ledger/test_infra/src/lib.rs

```diff
@@ -316,70 +307,3 @@ impl Ledger for MockLedger {
         unimplemented!()
     }
 }
-
```

Comment:
all of this was used to mimic a load balancer.
However we saw in real world deployment that the router did not work nicely with a load balancer.
So instead of leaving this extra complexity in the test, decided to remove it and focus the tests a bit more on the store behavior
```mermaid
flowchart  LR
A(Router) --> B(Load Balancer 1)
A --> C(Load Balancer 2)
A --> D(Load Balancer 3)
B --> F[Store A]
B --> G[Store B]
B --> H[Store C]
C --> I[Store D]
C --> J[Store E]
C --> K[Store F]
D --> L[Store G]
D --> M[Store H]
D --> N[Store I]
```

Reply:
Roger - IIUC, this is essentially dead code in our real deployments (no load balancer), so no need to keep maintaining this test code.